tests/platform_tests/test_reload_config.py

```diff
@@ -224,7 +225,8 @@ def test_reload_configuration_checks(duthosts, enum_rand_one_per_hwsku_hostname,
     if not duthost.get_facts().get("modular_chassis"):
         # Check if all containers have started
         assert wait_until(300, 10, 0, check_docker_status, duthost), (
-            "Not all Docker containers reached the 'fully started' state within 300 seconds "
+            "Not all Docker containers reached the 'fully started' state within the timeout. "
+            "Current status: '{}'.".format(check_docker_status(duthost))
```

Comment:
it does have failed reason in this assertion, run check_docker_status(duthost) again is not a good idea I think. Please remove this line